src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -22,11 +23,11 @@ public sealed class LicenseManager
     {
         private static readonly object s_selfLock = new object();
 
-        private static volatile LicenseContext s_context;
+        private static LicenseContext s_context;
         private static object s_contextLockHolder;
         private static volatile Hashtable s_providers;
         private static volatile Hashtable s_providerInstances;
-        private static readonly object s_internalSyncObject = new object();
+        private static object s_internalSyncObject = new object();
```

Comment:
Actually, no, I take that back, it's used elsewhere and needs to be initialized.  In which case it's also better for it to be readonly.  So let's just revert the changes here.